apps/chat/src/pages/index.tsx

```diff
@@ -348,6 +348,9 @@ export const getServerSideProps: GetServerSideProps = async ({
     themesHostDefined: !!process.env.THEMES_CONFIG_HOST,
     customRenderers: customRenderers || [],
     allowVisualizerSendMessages: !!process.env.ALLOW_VISUALIZER_SEND_MESSAGES,
+    isAdminUser: !!(session?.user.dial_roles
```

Comment:
I suggest to add this to auth store reducer, because it's more state of user, not setting of app